ctapipe/reco/shower_processor.py

```diff
@@ -111,43 +109,21 @@ def reconstruct_geometry(
         # Reconstruct the shower only if all shower criteria are met
         if len(hillas_dict) > 2:
 
-            array_pointing = SkyCoord(
-                az=event.pointing.array_azimuth,
-                alt=event.pointing.array_altitude,
-                frame=AltAz(),
-            )
-
-            telescopes_pointings = {
-                tel_id: SkyCoord(
-                    alt=event.pointing.tel[tel_id].altitude,
-                    az=event.pointing.tel[tel_id].azimuth,
-                    frame=AltAz(),
-                )
-                for tel_id in hillas_dict
-            }
-
-            result = self.reconstructor.predict(hillas_dict,
-                                                self.subarray,
-                                                array_pointing,
-                                                telescopes_pointings)
-
-            return result
+            self.reconstructor(event)
```

Comment:
don't you have to assign this to `event.dl2.stereo.geometry[self.reconstructor.__name__]`?  Otherwise the result doesn't go anywhere, unless i'm missing something.

Reply:
that is done inside the call method